mysql-ballerina/client.bal

```diff
@@ -161,10 +161,17 @@ public type Options record {|
     string serverTimezone?;
 |};
 
-# Possible options for SSL Mode.
+# Establish an encrypted connection if the server supports encrypted connections, falling back to an unencrypted
+# connection if an encrypted connection cannot be established.
 public const SSL_PREFERRED = "PREFERRED";
+# Establish an encrypted connection if the server supports encrypted connections. The connection attempt fails if
+# an encrypted connection cannot be established.
 public const SSL_REQUIRED = "REQUIRED";
+# Like SSL_REQUIRED, but additionally verify the server Certificate Authority (CA) certificate against the configured
+# CA certificates. The connection attempt fails if no valid matching CA certificates are found.
 public const SSL_VERIFY_CA = "VERIFY_CA";
+#Like SSL_VERIFY_CA, but additionally perform host name identity verification by checking the host name the client
```

Comment:
```suggestion
# Like SSL_VERIFY_CA, but additionally perform host name identity verification by checking the host name the client
```

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/157/commits/98bb69547c284bf2186bc2a14cbb622cd2fc34ad